src/frontend/components/SøknadsSteg/OmBarnet/Oppfølgningsspørsmål.tsx

```diff
@@ -38,11 +41,24 @@ const Oppfølgningsspørsmål: React.FC<{
     leggTilUtenlandsperiode: (periode: IUtenlandsperiode) => void;
     fjernUtenlandsperiode: (periode: IUtenlandsperiode) => void;
     utenlandsperioder: IUtenlandsperiode[];
-}> = ({ barn, skjema, leggTilUtenlandsperiode, fjernUtenlandsperiode, utenlandsperioder }) => {
+    leggTilBarnetrygdsperiode: (periode: IBarnetrygdsperiode) => void;
+    fjernBarnetrygdsperiode: (periode: IBarnetrygdsperiode) => void;
+    registrerteEøsBarnetrygdsperioder: Felt<IBarnetrygdsperiode[]>;
+}> = ({
+    barn,
+    skjema,
+    leggTilUtenlandsperiode,
+    fjernUtenlandsperiode,
+    utenlandsperioder,
+    leggTilBarnetrygdsperiode,
+    fjernBarnetrygdsperiode,
+    registrerteEøsBarnetrygdsperioder,
+}) => {
     const intl = useIntl();
     const { erÅpen, toggleModal } = useModal();
     const { erEøsLand } = useEøs();
     const { toggles } = useFeatureToggles();
+    const { erÅpen: barnetrygdsmodalErÅpen, toggleModal: toggleBarnetrygdsmodal } = useModal();
```

Comment:
Når vi legger til enda er useModal ville jeg renamet den andre og så man kan skille på de to 😊